common/types/layer.go

```diff
@@ -14,8 +14,8 @@ const (
 	// LayerIDSize in bytes.
 	LayerIDSize = 4
 
-	// GenesisBeacon is the hex value of the beacon used during genesis.
-	GenesisBeacon = "0xaeebad4a796fcc2e15dc4c6061b45ed9b373f26adfc798ca7d2d8cc58182718e" // sha256("genesis")
+	// BootstrapBeacon is the hex value of the beacon used during genesis.
+	BootstrapBeacon = "0x333c04dd151a2a6831c039cb9a651df29198be8a04e16ce861d4b6a34a11c954" // sha256("bootstrap")
```

Comment:
i would rather leave it as 32 zeros

Reply:
all zeros (types.EmptyBeacon) is used in some places as error condition. at this point i don't want to spend more time on this. i am assuming when we are getting ready for mainnet, this will change again.